spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config/README.md

```diff
@@ -32,15 +32,15 @@ or
 
```

Comment:
Line 1: Can we change the title of the document to something like **Azure App Configuration for Spring Cloud**?

Line 13 & 23: We should add a note when to use which dependency (eg., use the -web one in a web application...)

Reply:
For Line 13 & 23 I was thinking the same thing, but couldn't figure out how to word it. In the sections below I mention when to use each one.